assets/js/checkout/checkout.js

```diff
@@ -5,7 +5,7 @@ if (storefrontAppDependencies != undefined) {
     storefrontAppDependencies.push(moduleName);
 }
 angular.module(moduleName, ['credit-cards', 'angular.filter'])
-    .controller('checkoutController', ['$rootScope', '$scope', '$window', 'cartService',
+    .controller('CheckoutController', ['$rootScope', '$scope', '$window', 'cartService',
```

Comment:
We are using camelCase everywhere for the controller names, so I think this change is redundant

Reply:
Actually, CamelCase starts with first upper letter. Your example is lowerCamelCase.
EsLint is give me that advice. :)
But okay, i'll fix it.
By the way https://github.com/EmmanuelDemey/eslint-plugin-angular/blob/master/docs/rules/controller-name.md

Reply:
It depends :). I heard camelCase mainly for lowerCamelCase by default (e.g. camelCase vs PascalCase).
https://en.wikipedia.org/wiki/Camel_case.
For controller names - agree, it could be alternatives. But we are using camelCase everywhere, so let's follow this rule.